packages/rest/test/integration/http-handler.integration.ts

```diff
@@ -272,7 +264,83 @@ describe('HttpHandler', () => {
         });
     });
 
+    it('rejects unsupported request body', () => {
+      logErrorsExcept(415);
+      return client
+        .post('/show-body')
+        .set('content-type', 'application/xml')
+        .send('<key>value</key>')
+        .expect(415, {
+          error: {
+            message: 'Content-type application/xml is not supported.',
+            name: 'UnsupportedMediaTypeError',
+            statusCode: 415,
+          },
+        });
+    });
+
+    let bodyParamControllerInvoked = false;
+    it('rejects over-limit request form body', () => {
+      logErrorsExcept(413);
+      return client
+        .post('/show-body')
+        .set('content-type', 'application/x-www-form-urlencoded')
+        .send('key=' + givenLargeRequest())
+        .expect(413, {
+          error: {
+            message: 'request entity too large',
+            name: 'Error',
+            statusCode: 413,
+          },
+        })
+        .catch(catchEpipeError)
+        .then(() => expect(bodyParamControllerInvoked).be.false());
+    });
+
+    it('rejects over-limit request json body', () => {
+      logErrorsExcept(413);
+      return client
+        .post('/show-body')
+        .set('content-type', 'application/json')
+        .send({key: givenLargeRequest()})
+        .expect(413, {
+          error: {
+            message: 'request entity too large',
+            name: 'Error',
+            statusCode: 413,
+          },
+        })
+        .catch(catchEpipeError)
```

Comment:
When I read this line, I have no idea what is the intent. What happens when `EPIPE` error is caught? What happens when a different error was thrown?

Also I believe `E` in `EPIPE` is an abbreviation for `Error` (`EPIPE` = error pipe), thus `EpipeError` feels rather redundant to me ("error pipe error").

I am proposing to rename `catchEpipeError` to `ignorePipeError`.